packages/column-views/stories/facets/detrital-zircon.stories.ts

```diff
@@ -109,3 +124,22 @@ export function DetritalZirconCompilation(defaultArgs) {
     ])
   );
 }
+
+function getLayer(props) {
```

Comment:
Seems like there might be a better way to arrange this that could better use react's capabilities